dataset_preprocessing/tutorials/CONUS404/01_nwm_benchmark_preprocessing.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
And this notebook is an update to this older one? https://github.com/hytest-org/hytest/blob/main/dataset_preprocessing/tutorials/nwm/nwm_rechunking.ipynb

Has any code changed or is it just adding instructional material?

Reply:
@amsnyder, thats merely my reference working book and will be deleted upon completion of the preprocessing notebook.

Reply:
I'm not sure I understand...do we need two copies on Github if it's the same thing?